CMake error with Xcode generator

When using Xcode generator the following error occurs with Xcode 9.4.1 and CMake 3.10.1 during cmake file generation:

```
% cmake -GXcode ../visp
CMake Error in CMakeLists.txt:
  Evaluation file to be written multiple times with different content.  This
  is generally caused by the content evaluating the configuration type,
  language, or location of object files:

   /Users/fspindle/soft/ViSP/ViSP-dev/test-pr/visp-fspindle-build-osx/VISPGenerateConfigScript.info.cmake
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -933,7 +933,9 @@ endif(BUILD_PACKAGE)
 # Usage:
 #    visp-config --cflags ...
 #----------------------------------------------------------------------
-include(cmake/VISPGenerateConfigScript.cmake)
+if(NOT CMAKE_GENERATOR MATCHES "Xcode")
+  include(cmake/VISPGenerateConfigScript.cmake)
+endif()
 
 #----------------------------------------------------------------------
 # Propagation in sub dirs to build demo, example, test, tutorial

